ewmap.py

```diff
@@ -1924,6 +1924,17 @@ async def loop(cmd):
 		response = "You need to be on the edge of the map to !loop through it. Try a district bordering an outskirt, the ferry, or Slime's End Cliffs."
 		return await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 	else:
+		mutation_data = EwMutation(id_mutation=ewcfg.mutation_id_landlocked, id_user=cmd.message.author.id, id_server=cmd.message.guild.id)
+		
+		if len(mutation_data.data) > 0:
+			time_lastuse = int(mutation_data.data)
```

Comment:
you don't actually set this data anywhere. you have to set it with the current time, after a !loop is actually initialized

Reply:
this is fine now